components/raftstore/src/store/fsm/peer.rs

```diff
@@ -266,6 +267,11 @@ where
         );
         HIBERNATED_PEER_STATE_GAUGE.awaken.inc();
         let (tx, rx) = mpsc::loose_bounded(cfg.notify_capacity);
+        coprocessor_host.on_peer_created(
```

Comment:
Do you need a test to verify this?

Reply:
According to @BusyJay , I will remove this observer, and turn to `on_role_change` observer